sycl/doc/extensions/supported/sycl_ext_oneapi_local_memory.asciidoc

```diff
@@ -195,25 +194,7 @@ The example above creates a kernel with four work-groups, each containing 32
 work-items. An `int[64]` object is defined in group-local memory, and
 each work-item in the work-group obtains a `multi_ptr` to the same allocation.
```

Comment:
I think it would be useful to explain here that the elements of the array initialized to zero, which might not be obvious to an average C++ user.  (The array is aggregate initialized, but there are not any `args` in the initializer list.  This causes the array elements to be value initialized, which means they are set to zero.)

Reply:
Thanks, that seems good.